htdocs/core/class/extrafields.class.php

```diff
@@ -305,7 +305,26 @@ private function create_label($attrname, $label='', $type='', $pos=0, $size=0, $
 				$params='';
 			}
 
-			$sql = "INSERT INTO ".MAIN_DB_PREFIX."extrafields(name, label, type, pos, size, entity, elementtype, fieldunique, fieldrequired, param, alwayseditable, perms, list, ishidden, fielddefault, fieldcomputed)";
+			$sql = "INSERT INTO ".MAIN_DB_PREFIX."extrafields(";
+			$sql.= " name,";
+			$sql.= " label,";
+			$sql.= " type,";
+			$sql.= " pos,";
+			$sql.= " size,";
```

Comment:
@eldy 

It is an oversight because the value is always present!

$sql.= " ".$conf->entity.",";